source/extensions/common/wasm/wavm/wavm.cc

```diff
@@ -221,6 +232,71 @@ struct Wavm : public WasmVm {
 
   void getInstantiatedGlobals();
 
+  void getFunction(absl::string_view functionName, WasmCall0Void* f) override {
```

Comment:
Perhaps we could wrap this repetition up in a macro?

Reply:
Done.